### DISCUSSION TITLE
Parallels CTR Ubuntu Kinetic @ Arm64 CPU

### DISCUSSION THREAD

**Discussion by rastislavcore** [catchthatrabbit/coreminer#29] [Operating System]
### OS

Ubuntu Kinetic Desktop 22.10

With:
- Coreminer v0.19.60
- mine.sh script
- Autorun after the start
- Logs rotation
- Dark design & Wallpaper

### Software

Parallels Desktop 17 @ macOS Monterey 12.3.1

### Hardware

Apple M1 chip (Arm64)

### Virtual hardware

- 6x CPU
- 2048 MB Memory
- 10 GB Disk

### Download

Download from [Google Drive](https://drive.google.com/file/d/1V_7ddmxuJYIFkh-wz8ZwLxuN8VKt6Jpe/view?usp=sharing)

- Filesize: 3.15 GB
- Compression: XZ (hard - takes longer to uncompress)
- Username: `ubuntu`
- Password: `ubuntu`

**Comment by simanuel** [comment]
I would love to see that for UTM! (https://mac.getutm.app/)

**Reply by rastislavcore** [reply]
> I would love to see that for UTM! (https://mac.getutm.app/)

Please, open "wish" here https://github.com/catchthatrabbit/coreminer/discussions/categories/wishlist

And as soon as possible, we will do our best ;)
